### DISCUSSION TITLE
UpdateHub v2.3.4

### DISCUSSION THREAD

**Discussion by onyx-nxt** [NexovaDev/UpdateHub#953] [Announcements]
## Version
`2.3.4`

# Changelog
- Better Windows Update error handling
- Reduce UAC prompts to one per session
- Fix wrong app showing when clicking an item in the recently updated page list
- Fix app freezing when loading large lists
- Fix additional metadata not saving to the recently updated collection
- Made icon styling more consistent
- Fix minor grammar mistakes
- And more bug fixes!

# How to update
Click the Update Available button in the sidebar. (if you don't see it, force close the app by pressing the shift key, clicking the X icon in the windows title bar, and then relaunching the app.)

<img src="https://github.com/NexovaDev/UpdateHub/assets/71038229/d190f605-897d-4e6f-b147-fcccb07bdfe2" width="584" />
Then wait for the update file to download, and it will install automatically. You now have the latest version!

### VirusTotal scan
This app is not a virus, trust me. If you want, here's a [VirusTotal scan](https://www.virustotal.com/gui/file/54081ea02757300b4dd2967bf31a33dd92b834aa6666e79eea8bc22b6fbf364a "VirusTotal Scan"). As you can see, no anti-virus software has flagged it as a virus. The executable below's SHA-256 Hash is `54081EA02757300B4DD2967BF31A33DD92B834AA6666E79EEA8BC22B6FBF364A`, which matches the VirusTotal scan.

### Remember, if you encounter any bugs in UpdateHub, report them so we can fix them!

The one-click installer is attached below.

### SHA256
`54081EA02757300B4DD2967BF31A33DD92B834AA6666E79EEA8BC22B6FBF364A`

<hr /><em>This discussion was created from the release <a href='https://github.com/NexovaDev/UpdateHub/releases/tag/v2.3.4'>UpdateHub v2.3.4</a>.</em>

**Comment by Aardvogel** [comment]
This program would be hugely improved if you allowed users to specify additional  program installation folders as not everyone accepts the default installation to C:\Program Files etc. The bulk of my program files are installed on a separate hard drive not on my Windows drive, consequently I hardly pick up any updates.

Regards, Mike

**Reply by onyx-nxt** [reply]
You can vote for this feature [here](https://github.com/NexovaDev/UpdateHub/discussions/897). Adding custom install directories will take some modifications to the code as UpdateHub uses hardcoded paths to function properly. The poll is for me to see what most people prefer

**Reply by Aardvogel** [reply]
No this was not what I was asking for. It has nothing to do with where I want UpdateHub to be installed. What I was asking for was for UpdateHub to scan the folders where I have installed the bulk of my programs. I need UpfateHub to scan F:\Program Files and F:\Program Files (x86) in addition to the default C:\ folders.

Regards, Mike

**Comment by enduser1** [comment]
Win10 pro, if I install as a standard user and run the installer with admin rights it generates an error at the very end and install never completes. If I login as admin the app installs and runs correctly, but this defeats the purpose of not running PC with full admin right all the time, and prevents me as a standard user from running the app with admin rights to update. I also don't see a make this available for all users option to run this updater. All other apps that I install as standard user and provide admin rights, those apps install and run correctly.

![Capture](https://github.com/NexovaDev/UpdateHub/assets/35976366/ab36d7a1-30a6-4a02-882b-ae0e634d8f79)
